### PR TITLE
Production Merge

### PR DESCRIPTION
**Pull request by lespaceman**

### CODE REVIEW

File: src/data/activityData.json [hunk 1]
```diff
@@ -1,251 +1,245 @@
...
+    "bookingContactNumber": 8547603199,
+    "activityDescription": "Despite being the smallest national park in Kerala, the splendor of the shola forests in\r\nPampadum Shola National Park will mesmerize everyone’s mind. The Log House provides a\r\njungle camp package situated in the heart of shola forests. It costs 3000 rupees for 2 people."
+  },
+  {
+    "activitiesId": "ACT010",
+    "activityName": "Eravikulam National Park Jungle Camp",
+    "activityRouteId": "R1",
+    "bannerImg": "https://cdn.hellomunnar.in/activities/Eravikulam_National_Park_Jungle_Camp.jpg",
+    "activityOpenTime": "3 : 00 PM",
+    "activityCloseTime": "10.00 AM",
```
Comment (athul) [inline src/data/activityData.json:145]: There seems to be some descrepancy here on the closetime. Shouldn't it be `10 : 00`

**Review by athul — COMMENTED** [review]
There seems to be a non uniformity in defining the time. Shouldn't that be changed?

File: src/data/activityData.json (hunk 1 above)
Comment (lespaceman) [reply src/data/activityData.json:145]: @athul yes, the data team is on it. Merge it for now.

Comment (athul) [reply src/data/activityData.json:145]: Wokay

**Review by athul — APPROVED** [review]

**Pull request merged**